themes/gatsby-theme-blog/src/components/post-footer-content.js

```diff
@@ -24,15 +24,15 @@ export default ({ previous, next }) => (
       >
```

Comment:
I thought we were moving this out to the post-footer? Or maybe I'm misremembering.

Reply:
I don't remember that, but I don't have notes on it in my doc... We were going for the idea of consolidating content components for people to shadow, and then walking up in complexity from there.